Sooty.py

```diff
@@ -426,10 +426,18 @@ def repChecker():
             response = requests.get(BAD_IPS_URL)
             if response.status_code == 200:
                 result = response.json()
-
-                sc = result['Score']['ssh']
+                sc = 0
+                for each in result['Categories']:
+                    temp = each
+                    sc += result['Score']['temp']
```

Comment:
Calculate the score for all the categories it was reported under and sum it all.